addons/website_event_meet/static/src/js/website_event_create_meeting_room_button.js

```diff
@@ -15,23 +15,20 @@ publicWidget.registry.websiteEventCreateMeetingRoom = publicWidget.Widget.extend
     //--------------------------------------------------------------------------
 
     _onClickCreate: async function () {
-        if (!this.$createModal) {
+        if (!this.createModalEl) {
             const langs = await rpc("/event/active_langs");
 
-            this.$createModal = $(renderToElement(
-                'event_meet_create_room_modal',
-                {
+            this.createModalEl = renderToElement("event_meet_create_room_modal", {
                     csrf_token: odoo.csrf_token,
                     eventId: this.$el.data("eventId"),
                     defaultLangCode: this.$el.data("defaultLangCode"),
```

Comment:
`.data(...)` is jQuery.

Reply:
Let me look into it. How this could have been missed :sweat_smile: